### PR TITLE
MP-8784: Added the following types to allnoxtype entity

### PR DESCRIPTION
**Pull request by originalethanhunt**
Color
HtmlField
InternetDomainField
IpAddressField
JsonField
JwtTokenField
MonthField
NuidField

### CODE REVIEW

File: src/Nox.Types.Abstractions/Enums/NoxType.cs [hunk 1]
```diff
@@ -85,7 +85,7 @@ public enum NoxType : uint
     [SimpleType(typeof(bool))]
     Boolean = 2157507194,
 
-    [SimpleType(typeof(string))]
+    [SimpleType(typeof(byte[]))]
```
Comment (rochar) [inline src/Nox.Types.Abstractions/Enums/NoxType.cs:88]: @gjorgji-grgovski-authoritypartners we will keep Color as string in dtos?

Comment (andresharpe) [reply src/Nox.Types.Abstractions/Enums/NoxType.cs]: I think we should keep this simple and persist as a string in well known hex format, eg.:

#CD5C5C

Comment (andresharpe) [reply src/Nox.Types.Abstractions/Enums/NoxType.cs]: so it'll map to varchar(7) in sql server

Comment (originalethanhunt) [reply src/Nox.Types.Abstractions/Enums/NoxType.cs]: it's ARGB so I guess that will be varchar(9).

Comment (rochar) [reply src/Nox.Types.Abstractions/Enums/NoxType.cs]: so lets make this string again, so we can merge this PR

@gjorgji-grgovski-authoritypartners  is taking care of creating a tickets to refactor Color Value Object to be string and a converter to varchar(9)

**Review by rochar — CHANGES_REQUESTED** [review]

**Comment by rochar** [comment]
Please also update it from main

**Review by rochar — APPROVED** [review]

**Pull request merged**